src/overlays/actors/ovl_En_Test6/z_en_test6.c

```diff
@@ -35,50 +53,1278 @@ const ActorInit En_Test6_InitVars = {
+        sin = Math_SinS(temp_s3) * this->unk_150;
+        Matrix_InsertTranslation(cos, phi_f24, sin, MTXMODE_NEW);
+        Matrix_InsertXRotation_s(0x4000, MTXMODE_APPLY);
+        Matrix_Scale(0.8f, 0.8f, 0.8f, MTXMODE_APPLY);
+        Matrix_InsertZRotation_s(phi_s2, MTXMODE_APPLY);
+
+        gSPMatrix(this->unk_148++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gDPSetPrimColor(this->unk_148++, 0, 0xFF, 28, 28, 28, 255);
+        gDPSetEnvColor(this->unk_148++, 255, 255, 255, 255);
+        gDPSetRenderMode(this->unk_148++, G_RM_FOG_SHADE_A, G_RM_AA_ZB_OPA_SURF2);
+        gSPDisplayList(this->unk_148++, gameplay_keep_DL_07AFB0);
+
+        temp_s4 += 0x1000;
+        cos = Math_CosS(temp_s4) * this->unk_150;
+        sin = Math_SinS(temp_s4) * this->unk_150;
+        Matrix_InsertTranslation(cos, phi_f24, sin, MTXMODE_NEW);
+        Matrix_InsertXRotation_s(0x4000, MTXMODE_APPLY);
+        Matrix_Scale(0.8f, 0.8f, 0.8f, MTXMODE_APPLY);
+        Matrix_InsertZRotation_s(-phi_s2, MTXMODE_APPLY);
+
+        gSPMatrix(this->unk_148++, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gDPSetPrimColor(this->unk_148++, 0, 0xFF, 28, 28, 28, 255);
+        gDPSetEnvColor(this->unk_148++, 255, 255, 255, 255);
+        gDPSetRenderMode(this->unk_148++, G_RM_FOG_SHADE_A, G_RM_AA_ZB_OPA_SURF2);
+        gSPDisplayList(this->unk_148++, gameplay_keep_DL_07AFB0);
+
+        phi_f24 -= this->unk_14C;
+        phi_s2 += this->unk_27E;
+    }
+
+    POLY_OPA_DISP = this->unk_148;
+
+    for (i = 0; i < ARRAY_COUNT(D_80A94910); i++) {
+        if (D_80A94910[i].unk_04 > 0.0f) {
+            D_80A94910[i].unk_14(this, globalCtx, &D_80A94910[i]);
+        }
+    }
+
+    CLOSE_DISPS(globalCtx->state.gfxCtx);
+}
+
+void func_80A9369C(Actor* thisx, GlobalContext* globalCtx2) {
+    EnTest6* this = THIS;
+    GlobalContext* globalCtx = globalCtx2;
+    f32 temp_f20;
+    f32 temp_f22;
+    f32 phi_f26;
+    s16 phi_s2;
+    s16 sp78;
+    s32 i;
+    Player* player = GET_PLAYER(globalCtx);
+    s32 pad;
+
+    if (this) {}
+    OPEN_DISPS(globalCtx->state.gfxCtx);
+
+    this->unk_148 = POLY_OPA_DISP;
+    this->unk_27C += (s16)this->unk_154;
+    this->unk_27E = this->unk_27C * 2;
+    sp78 = (globalCtx->state.frames & 0x3C) * 1024;
+    phi_s2 = this->unk_27C + 0x4000;
+
+    switch (player->transformation) {
+        default:
+            phi_f26 = player->actor.world.pos.y + 40.0f;
+            break;
+
+        case PLAYER_FORM_DEKU:
+            phi_f26 = player->actor.world.pos.y + 40.0f;
+            break;
+
+        case PLAYER_FORM_GORON:
+            phi_f26 = player->actor.world.pos.y + 40.0f;
+            break;
+
+        case PLAYER_FORM_ZORA:
+            phi_f26 = player->actor.world.pos.y + 40.0f;
+            break;
+
+        case PLAYER_FORM_FIERCE_DEITY:
+            phi_f26 = player->actor.world.pos.y + 40.0f;
+    }
```

Comment:
Any reason to omit the break;?
```suggestion
        case PLAYER_FORM_FIERCE_DEITY:
            phi_f26 = player->actor.world.pos.y + 40.0f;
            break;
    }
```